cron/.env-dist

```
@@ -0,0 +1,3 @@
+GIT_REPO=https://github.com/abes-esr/abes-labo-movies
```

peut-être y ajouter les 3 autres variables clées :

```suggestion
GIT_REPO=https://github.com/abes-esr/abes-labo-movies
NEO4J_BASE_URL=http://127.0.0.1:7474
NEO4J_USER=neo4j
NEO4J_PASSWORD=secret
```